pam.d/common-session

```diff
@@ -10,5 +10,4 @@
 session	substack	common-session-noninteractive
 
 session	optional	pam_mkhomedir.so
-session	optional	pam_systemd.so
```

Comment:
Wouldn't this turn off system for interactive sessions?

Reply:
No, because everything that runs for non-interactive sessions also runs for interactive ones.